### ISSUE TITLE
Measure the computing time of particle interpolation

### ISSUE THREAD

**Issue by hlokavarapu** [geodynamics/aspect#1565]
At the moment, the time taken for the particle interpolation algorithm is not being measured. This piece of information will be important in better understanding the cost of different interpolation algorithms, the scaling properties, the time complexity as a function of number of particles, and etc.

**Comment by hlokavarapu** [comment]
@gassmoeller 

After adding call to `computing_timer.enter_section("   Particles: Interpolate")`  in the function `void Simulator<dim>::interpolate_particle_properties (const AdvectionField &advection_field)` the timing summary is such that:

```
+---------------------------------------------+------------+------------+
| Total wallclock time elapsed since start    |      1.48s |            |
|                                             |            |            |
| Section                         | no. calls |  wall time | % of total |
+---------------------------------+-----------+------------+------------+
| Assemble Stokes system          |         2 |    0.0433s |       2.9% |
| Assemble composition system     |         2 |     0.209s |        14% |
| Assemble temperature system     |         2 |    0.0716s |       4.8% |
| Build Stokes preconditioner     |         1 |    0.0248s |       1.7% |
| Build composition preconditioner|         2 |    0.0134s |       0.9% |
| Particles: Interpolate          |         4 |    0.0268s |       1.8% |
| Solve Stokes system             |         2 |      0.07s |       4.7% |
| Solve composition system        |         2 |   0.00376s |      0.25% |
| Initialization                  |         1 |    0.0264s |       1.8% |
| Particles: Advect               |         4 |     0.287s |        19% |
| Particles: Exchange ghosts      |         3 |   0.00322s |      0.22% |
| Particles: Generate             |         1 |    0.0436s |       2.9% |
| Particles: Initialization       |         1 |  0.000129s |         0% |
| Particles: Initialize properties|         1 |    0.0458s |       3.1% |
| Particles: Sort                 |         4 |    0.0215s |       1.4% |
| Particles: Update properties    |         2 |      0.34s |        23% |
| Postprocessing                  |         2 |     0.695s |        47% |
| Setup dof systems               |         1 |     0.178s |        12% |
| Setup initial conditions        |         1 |       0.1s |       6.8% |
+---------------------------------+-----------+------------+------------+
```

Is there a method of controlling where `Particles: Interpolate` is listed in the summary?

**Comment by gassmoeller** [comment]
I though it would be sorted in the order the `std::map<std::string,Section>` would sort the entries, which should be alphabetical. In your post above you have a space in front of the `Particles` do you add the same in the code? What happens if you remove the space?

**Comment by hlokavarapu** [comment]
Super simple, thanks Rene!

```
+---------------------------------------------+------------+------------+
| Total wallclock time elapsed since start    |      1.42s |            |
|                                             |            |            |
| Section                         | no. calls |  wall time | % of total |
+---------------------------------+-----------+------------+------------+
| Assemble Stokes system          |         2 |    0.0414s |       2.9% |
| Assemble composition system     |         2 |     0.195s |        14% |
| Assemble temperature system     |         2 |    0.0686s |       4.8% |
| Build Stokes preconditioner     |         1 |    0.0245s |       1.7% |
| Build composition preconditioner|         2 |    0.0122s |      0.86% |
| Solve Stokes system             |         2 |    0.0406s |       2.9% |
| Solve composition system        |         2 |   0.00313s |      0.22% |
| Initialization                  |         1 |    0.0267s |       1.9% |
| Particles: Advect               |         4 |     0.286s |        20% |
| Particles: Exchange ghosts      |         3 |   0.00306s |      0.22% |
| Particles: Generate             |         1 |    0.0422s |         3% |
| Particles: Initialization       |         1 |  0.000125s |         0% |
| Particles: Initialize properties|         1 |    0.0461s |       3.3% |
| Particles: Interpolate          |         4 |    0.0251s |       1.8% |
| Particles: Sort                 |         4 |    0.0229s |       1.6% |
| Particles: Update properties    |         2 |      0.33s |        23% |
| Postprocessing                  |         2 |     0.685s |        48% |
| Setup dof systems               |         1 |     0.179s |        13% |
| Setup initial conditions        |         1 |    0.0978s |       6.9% |
+---------------------------------+-----------+------------+------------+
```

**Comment by bangerth** [comment]
Will be fixed by #1573.

**Comment by gassmoeller** [comment]
closed by #1573
